frontend/src/components/navbar/Navbar.scss

```diff
@@ -1,9 +1,58 @@
 nav.navbar {
-  height: 4em;
   background-color: #bbffff;
   padding: 0.5em;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 
   img.navbar__logo {
-    height: 100%;
+    height: 3em;
+  }
+
+  a {
+    text-decoration: none;
+    color: #00aeff;
+    padding-left: 1em;
+    padding-right: 0.5em;
+  }
+
+  button.menu-toggle {
+    display: none;
+    border: 2px solid #00aeff;
+    background: transparent;
+    padding: 0.25em 0.5em;
+    border-radius: 4px;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  nav.navbar {
+    img.navbar__logo {
+      height: 2em;
+    }
+
+    .menu-items {
+      display: flex;
+      flex-direction: column;
+      position: absolute;
+      top: 3em;
+      width: 100%;
+      background-color: #bff;
+      left: 0;
+      align-items: center;
+
+      &.menu-collapsed {
+        display: none;
+      }
+
+      a.menu-link {
+        padding: 0.5em;
+      }
+    }
+
+    button.menu-toggle {
+      display: block;
+      margin-right: 0.5em;
+    }
```

Comment:
Did we conclude the conversation about whether or not we were going to use a styling library?